src/apps/TicTacToe.js

```diff
@@ -0,0 +1,148 @@
+import { useState } from "react";
+import styled from "styled-components";
+import { InstantApp } from "./types";
```

Comment:
Is this being used, since the file is `.js`?

Reply:
Nice catch, I was on my way to write to change it into typescript but left it for later. Prettier should have removed it.